js/src/Ice/ConnectionI.js

```diff
@@ -591,6 +595,16 @@ export class ConnectionI {
             ++count;
         }
 
+        if (messages != null) {
+            for (const message of messages) {
+                if (message.outAsync !== null) {
```

Comment:
messages is a list of outgoing requests for which we've received replies. How can outAsync be null?

Reply:
Yes no need for this null check.